tests/tools/oesign-engine/test-enclave/CMakeLists.txt

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+cmake_minimum_required(VERSION 3.11)
+
+project("oesign-test" LANGUAGES C CXX)
+
+set(CMAKE_CXX_STANDARD 11)
+
+add_subdirectory(enclave)
+add_subdirectory(host)
```

Comment:
Can this be simplified greatly? All you really need for this test is to compile the enclave binary and you don't need any of the host pieces here at all, do you?

Reply:
We run the host side in the test in order to verify the signing process by loading the enclave.